src/index.ts

```diff
@@ -66,34 +124,49 @@ const getAuthTokens = (): IAuthTokens | undefined => {
   return
 }
 
-export const getRefreshToken = (): Token | undefined => {
-  const tokens = getAuthTokens()
-  return tokens ? tokens.refreshToken : undefined
-}
-export const getAccessToken = (): Token | undefined => {
-  const tokens = getAuthTokens()
-  return tokens ? tokens.accessToken : undefined
-}
+/**
+ * Checks if the token is undefined, has expired or is about the expire
+ *
+ * @param {string} token - Access token
+ * @returns Whether or not the token is undefined, has expired or is about the expire
+ */
 const isTokenExpired = (token: Token): boolean => {
   if (!token) return true
-  const expin = getExpiresInFromJWT(token) - EXPIRE_FUDGE
-  return !expin || expin < 0
```

Comment:
It was named "expires in" because it's not a date it's how many seconds until the token is expired

Reply:
right, how about I rename the variable to `expiresIn` then?
I'll also rename `getExpiration` to `getExpiresIn` to make it more consistent

Reply:
Sure